### PR TITLE
Update CUDA_VERSION to 9.1

### PR DESCRIPTION
**Pull request by AswathySK**
The cuda path (C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.0) to v9.1 in Adoptium playbook.
Therefore update the CUDA_VERSION variable.
reference link: https://github.com/adoptium/infrastructure/pull/3587

### CODE REVIEW

**Review by steelhead31 — COMMENTED** [review]
I'll check that the CUDA toolkit has been updated on the Windows build machines prior to merging this PR. Once I've checked that, I'll approve.

**Review by steelhead31 — APPROVED** [review]
Both build machines look to have the new version of CUDA, so approving assuming all the checks run green.

**Review by andrew-m-leonard — APPROVED** [review]
lgtm

**Pull request merged**
